app/views/trucks/index.html.slim

```diff
@@ -9,14 +9,4 @@ table class="table"
       th scope="col" Status
       th scope="col" Actions
   tbody
-    - @trucks.each do |truck|
-      tr
-        td = truck.id
-        td = truck.license_plate
-        td = truck.capacity
-        td = truck.color
-        td = truck.service_date
-        td = truck.status
-        td
-          a(class="btn btn-primary" href=edit_truck_path(truck) role="button") Edit
-          a(class="btn btn-mini btn-danger" rel="nofollow" data-method="delete" data-confirm="Are you sure?" href=truck_path(truck)) Delete
+    = render partial: 'shared/truck'
```

Comment:
ehhhh i dont think this has brought us any value at all.
lets go over how we can refactor this later and i think undo this and bring it back to what it was before